### ISSUE TITLE
Images of entries not displaying

### ISSUE THREAD

**Issue by florian-levet** [NeuBIAS/bise#43]
It seems that a lot of images of the entries of bii.eu are not shown anymore, like if the link to them is missing

**Comment by PerrineGilloteaux** [comment]
ACTUALLY NONE of them is showing.
(only the one in your navigator cache...)

https://biii.eu/sites/default/files/ NOT accessible??
@miura it is likely related to the .htaccess?
It is actually the same with test.biii.eu.
We did not notice because images were in the cache.

**Comment by miura** [comment]
partially fixed: partially means that images in biii.eu should now be appearing - but images in test.biii.eu and for the local copies are not. I think the priority for the production site is higher so I did like this. I will investigate what is wrong with redirections. -- and will report more details.

**Comment by miura** [comment]
I cannot figure out a good way to create a condition applicable to both the production and the development environment. `.htaccess` is currently set differently in these two environments.

in test.biii.eu and in the local, following line redirects access to images to the production site.
``` 
RewriteRule ^(sites/default/files/.*\.(?:jpg|jpeg|JPG|png|PNG|gif|GIF|txt|csv|ico))$ https://biii.eu/$1 [L]
 ```

In the production site, this line is commented out to avoid the infinite loop of redirection. There should be a more clever way to set a condition but currently not working (failed this morning). Let's just for now go with this two different settings.

**Comment by vcaldas** [comment]
We can try adding the file to .gitignore. Then we can keep both copies separated.

**Comment by miura** [comment]
yes. So I added web/.htaccess and .gitignore itself to .gitignore in the production site. Just for us to keep this in mind!

**Comment by vcaldas** [comment]
I'm not sure that adding the .gitignore file in it is the right approach. Gitignore is a reserved file where we just add the files to be ignored. Otherwise, the updates will ignore the gitignore file. 😅

**Comment by miura** [comment]
I understand your concern - but let's keep it as a temporal solution. `production` branch is in the downstream of other branches, and updates in web/.htaccess is only ignored in biii.eu htacess. Limitation is that when this file is updated in local or other branches for different reason, it will not be reflected in the production site. We just need not to forget this.

**Comment by miura** [comment]
I let this issue opened again.

**Comment by miura** [comment]
here is the solution that works. I will PR this to `dev` later, and then to `master` and `production`. 

```
# If the request is to the files directory. 
RewriteCond %{REQUEST_URI} ^/sites/default/files/(.*)$ 
# and the environment is not production 
RewriteCond %{HTTP_HOST} !^(www.)?example.com$ [NC] 
RewriteCond %{REQUEST_FILENAME} !-f 
RewriteCond %{REQUEST_FILENAME} !-d 
# Redirect the request to the production server. 
RewriteRule ^sites/default/files/(.*)$ http://example.com/sites/default/files/$1 [L]
```

based on the description from [here](https://support.acquia.com/hc/en-us/articles/360007322714-Serving-files-from-production-on-development-environments)
